ffmpeg-docker/Dockerfile

```diff
@@ -0,0 +1,130 @@
+                    yasm \
+                    libpulse-dev \
+                    libxcb1-dev libxcb-shm0-dev   libxcb-xfixes0-dev \
+                    zlib1g-dev" && \
+        apt-get -yqq update && \
+        apt-get install -yq --no-install-recommends ${buildDeps}
+## x264 http://www.videolan.org/developers/x264.html
+RUN \
+        DIR=/tmp/x264 && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
+        curl -sL https://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-${X264_VERSION}.tar.bz2 | \
+        tar -jx --strip-components=1 && \
+        ./configure --prefix="${PREFIX}" --enable-shared --enable-pic --disable-cli && \
+        make && \
+        make install && \
+        rm -rf ${DIR}
+
+## ffmpeg https://ffmpeg.org/
+RUN  \
+        DIR=/tmp/ffmpeg && mkdir -p ${DIR} && cd ${DIR} && \
+        curl -sLO https://github.com/etherlabsio/FFmpeg/archive/n${FFMPEG_VERSION}.tar.gz && \
+        tar -zx --strip-components=1 -f n${FFMPEG_VERSION}.tar.gz 
+
+
+RUN \
+        DIR=/tmp/ffmpeg && mkdir -p ${DIR} && cd ${DIR} && \
+        ./configure \
+        --disable-debug \
+        --disable-doc \
+        --disable-ffplay \
+        --disable-avresample \
+        --enable-shared \
+        --disable-libopencore-amrnb \
+        --disable-libopencore-amrwb \
+        --enable-gpl \
+        --enable-libxcb \
+        --disable-libass \
+        --disable-libfreetype \
+        --disable-libvidstab \
+        --disable-libmp3lame \
+        --disable-libopenjpeg \
+        --disable-libopus \
+        --disable-libtheora \
+        --disable-libvorbis \
+        --disable-libvpx \
+        --disable-libx265 \
+        --disable-libxvid \
+        --enable-libx264 \
+        --disable-nonfree \
+        --disable-openssl \
+        --disable-libfdk_aac \
+        --disable-libkvazaar \
+        --disable-libaom --extra-libs=-lpthread \
+        --enable-postproc \
+        --enable-small \
+        --enable-version3 \
+        --enable-libpulse \
+        --extra-cflags="-I${PREFIX}/include" \
+        --extra-ldflags="-L${PREFIX}/lib" \
+        --extra-libs=-ldl \
+        --prefix="${PREFIX}" && \
+        make && \
+        make install && \
+        make distclean && \
+        hash -r && \
+        cd tools && \
+        make qt-faststart && \
+        cp qt-faststart ${PREFIX}/bin
+
+## cleanup
+RUN \
+        ldd ${PREFIX}/bin/ffmpeg | grep opt/ffmpeg | cut -d ' ' -f 3 | xargs -i cp {} /usr/local/lib/ && \
+        cp ${PREFIX}/bin/* /usr/local/bin/ && \
+        cp -r ${PREFIX}/share/ffmpeg /usr/local/share/ && \
+        LD_LIBRARY_PATH=/usr/local/lib ffmpeg -buildconf
+
+FROM        base AS release
+MAINTAINER  Julien Rottenberg <julien@rottenberg.info>
+
+ENV         LD_LIBRARY_PATH=/usr/local/lib
+
+COPY --from=build /usr/local /usr/local/
+
+# Let's make sure the app built correctly
+# Convenient to verify on https://hub.docker.com/r/jrottenberg/ffmpeg/builds/ console output
```

Comment:
@lijo-jose please take one more pass at reviewing this Dockerfile. Once you're confirmed, I can move this file to the `etherlabsio/ffmpeg` repo and have docker build a public image for this.